pkg/apis/aws/v1alpha1/accountclaim_types.go

```diff
@@ -25,9 +26,35 @@ type AccountClaimStatus struct {
 	// INSERT ADDITIONAL STATUS FIELD - define observed state of cluster
```

Comment:
Will this struct contain a `Phase` or `Status` field?  How about:
```suggestion
type ClaimStatus string
const (
   ClaimStatusPending ClaimStatus = "Pending"
   ClaimStatusReady    ClaimStatus = "Ready"
   ClaimStatusError       ClaimStatus = "Error"
)
...
type AccountClaimStatus struct {
    ...
    Status ClaimStatus
}
```

Reply:
Oh now I see this information is under `AccountClaimCondition` but - why an array of these? shouldn't there be one?

Reply:
The Conditions are a history of state so we append each change to the array